cg/services/bcl_convert_metrics_service/bcl_convert_metrics_service.py

```diff
@@ -77,3 +79,35 @@ def _create_bcl_convert_sequencing_metrics(
             sample_base_mean_quality_score=mean_quality_score,
             created_at=datetime.now(),
         )
+
+    @staticmethod
+    def create_sample_run_metrics(
+        sample_internal_id: str,
+        lane: int,
+        run_metrics_id: int,
+        metrics_parser: MetricsParser,
+        store: Store,
+    ) -> IlluminaSampleRunMetrics:
+        """Create sequencing metrics for all lanes in a flow cell."""
+
+        total_reads: int = metrics_parser.calculate_total_reads_for_sample_in_lane(
+            sample_internal_id=sample_internal_id, lane=lane
+        )
+        q30_bases_percent: float = metrics_parser.get_q30_bases_percent_for_sample_in_lane(
+            sample_internal_id=sample_internal_id, lane=lane
+        )
+        mean_quality_score: float = metrics_parser.get_mean_quality_score_for_sample_in_lane(
+            sample_internal_id=sample_internal_id, lane=lane
+        )
+        sample_id: int = store.get_sample_by_internal_id(sample_internal_id).id
+
+        return IlluminaSampleRunMetrics(
+            run_metrics_id=run_metrics_id,
+            sample_id=sample_id,
+            type=DeviceType.ILLUMINA,
+            flow_cell_lane=lane,
+            total_reads_in_lane=total_reads,
+            base_percentage_passing_q30=q30_bases_percent,
+            base_mean_quality_score=mean_quality_score,
+            created_at=datetime.now(),
+        )
```

Comment:
I think the pythonic way of relating objects is not to do it directly via the foreignkey. Instead you instantiante the object and then update it via the relationship. Some thing like this:
```suggestion
        sample: Sample = store.get_sample_by_internal_id(sample_internal_id)

        new_sample_metric = IlluminaSampleRunMetrics(
            type=DeviceType.ILLUMINA,
            flow_cell_lane=lane,
            total_reads_in_lane=total_reads,
            base_percentage_passing_q30=q30_bases_percent,
            base_mean_quality_score=mean_quality_score,
            created_at=datetime.now(),
        )
        new_sample_metric.sample = sample
        new_sample_metric.run_metric = run_metric
        return new_sample_metric
```

Reply:
Will revisit in next pr